source/accessible-media/production-captions.html.erb.md

```diff
@@ -89,35 +94,58 @@ formatted and timestamped.
 
 ### Caption quality
 
-Always provide the highest-quality captions that convey 100% accuracy.
+{::nomarkup}
+<%= ref :start %>
+{:/nomarkup}
+
 See [resources for writing quality captions](#resources-for-writing-quality-captions) for more
-information. Keep the following basic points in mind when writing
+information.
+
+{::nomarkup}
+<%= ref :middle %>
+{:/nomarkup}
+
+Always provide the highest-quality captions that convey 100% accuracy.
+Keep the following points in mind when writing
 captions:
 
 -   Ensure that there are no spelling errors. This includes the names of
     characters or speakers.
 -   Use conventional grammar rules. After end punctuation (period,
-    question mark, exclamation point, etc.), always begin a new caption.
+    question mark, exclamation point, etc.), always begin a new caption block.
 -   Do not edit unless you have a specific reason to do so (e.g., to
     achieve a specific reading level). Fillers such as "um," "ah," etc.,
     can be deleted to save reading time unless doing so alters the
     representation of the speaker.
 -   Do not censor: the captions should always accurately represent what
```

Comment:
the general rule is that captions should always reflect the audio.  if a swear is used in the audio track, that same swear should appear in the caption track.  if the swear is bleeped in the audio track, then the swear should be removed from the captions.  "Do not censor" refers more to the practice of editing "objectionable" language from captions even though that same language is in the audio track.  

but i think delving into censorship, whether it's author choice or otherwise, is out of scope for this tutorial.